week0001/counter.rb

```diff
@@ -0,0 +1,11 @@
+
+p "Start counting:" 
```

Comment:
No such requirement to print this line. Also rename your file like this: `fizz_buzz_congpine.rb`. Thanks!